torch/_torch_docs.py

```diff
@@ -1500,6 +1500,84 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input, other=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given input tensor data and weights (optional). The unbiased sample covariance 
+of the variables :math:`x` and :math:`y` is given by
+
+.. math::
+    \text{cov}_w(x,y) = \frac{\sum^{N}_{i = 1}(x_{i} - \bar{x})(y_{i} - \bar{y})}{N~-~1}
+
+where, :math:`\bar{x}` and :math:`\bar{y}` are the simple means of the variables respectively. 
+If :attr:`fweights` and/or :attr:`aweights` are provided, the unbaised weighted covariance 
+is calculated, which is given by
+
+.. math::
+    \text{cov}_w(x,y) = \frac{\sum^{N}_{i = 1}w_i(x_{i} - \mu_x^*)(y_{i} - \mu_y^*)}{\sum^{N}_{i = 1}w_i~-~1}
+
+where :math:`\mu_x^* = = \frac{\sum^{N}_{i = 1}w_ix_{i} }{\sum^{N}_{i = 1}w_i}` 
+is the weighted mean of the variable. 
+
+
+Args:
+    input (Tensor): A 1-D or 2-D tensor containing multiple variables and observations.
+        Each row of :attr:`input` represents a variable, and each column a
+        single observation of all those variables. Also see :attr:`rowvar` below.
+    other (Tensor, optional): An additional set of variables and observations.
+        :attr:`other` has the same form as that of :attr:`input` and can be used to provide
+        additional variables for each of the observations which would be considered along with those provided in
+        :attr:`input`. 
```

Comment:
This is a good description of "other," per other comment I think this should just explain that input and other are stacked before the computation if other is provided. The shapes of input, other, and the result should also be described line in your description on the PR.